trunk/NDHMS/Routing/Reservoirs/module_reservoir_utilities.F

```diff
@@ -269,25 +276,47 @@ subroutine read_persistence_netcdf_gage_id(ncid, lake_id_index, netcdf_array_nam
         integer, intent(in) :: ncid, lake_id_index
         character(len=*), intent(in) :: netcdf_array_name
         integer, intent(out) :: gage_id
-        integer, allocatable, dimension(:) :: temp_gage_id_array
+        character(len=15) :: gage_id_string, gage_id_string_trimmed
+        !integer, allocatable, dimension(:) :: temp_gage_id_array
```

Comment:
Git will track this change; no reason to leave the previous version here

Reply:
Removing this too.